src/coreclr/pal/src/map/map.cpp

```diff
@@ -2394,8 +2394,9 @@ void * MAPMapPEFile(HANDLE hFile, off_t offset)
         goto doneReleaseMappingCriticalSection;
     }
 
-    void* prevSectionEnd;
-    prevSectionEnd = (char*)loadedHeader + headerSize; // the first "section" for our purposes is the header
+    void* prevSectionEndAligned;
+    // the first "section" for our purposes is the header
+    prevSectionEndAligned = ALIGN_UP((char*)loadedHeader + headerSize, GetVirtualPageSize());
```

Comment:
The does not change how we map.  I just renamed `prevSectionEnd` and made it always aligned. I think it is more readable.